### ISSUE TITLE
GMVD test set or MultiviewX in Table 7?

### ISSUE THREAD

**Issue by sithu31296** [jeetv/GMVD_dataset#3]
Hello, I have some confusion about the results reported in Table 7. In the explanation, you said that the results are evaluated on MultiviewX dataset but later in the paragraph, it is changed to GMVD test set. And in the table caption, it is written as GMVD test set. Which is true?

**Comment by jeetv** [comment]
Hi @sithu31296,

Results reported in Table 7 are trained with GMVD dataset and evaluated on GMVD test set and Wildtrack (synthetic to real).
In paragraph, we want to convey that training happened on GMVD train set and we used MultiviewX as a validation set, while benchmarking in Table 7 we used GMVD test set and Wildtrack

**Comment by sithu31296** [comment]
Thank you
